### PR TITLE
BDCOM update support

### PR DESCRIPTION
**Pull request by hrtrd**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by hrtrd** [comment]
what happens?

**Comment by Jellyfrog** [comment]
Id suggest revert the changes on bdcom files. And add new ones with --variant specified.

File: includes/definitions/discovery/bdcom.yaml [hunk 1]
```diff
@@ -1,13 +1,127 @@
...
+                 - ponIfIndex
+                 - gponOltPonPortPortActiveOnuNum
+                 - gponOltPonPortPortInactiveOnuNum
     processors:
         data:
             -
-                oid: bdpmCPUTotal5min
-                num_oid: '.1.3.6.1.4.1.3320.9.109.1.1.1.1.5.{{ $index }}'
-                index: 0
+                oid: nmscardCPUUtilization
```
Comment (murrant) [inline includes/definitions/discovery/bdcom.yaml:29]: According the the tests, it is using HOST-RESOURCES-MIB instead of this...  Maybe a mib failure?

**Comment by PipoCanaja** [comment]
Yep, @hrtrd 
Could you restore the original bdcom test files and create a new variant for your tests ?
(so we end up with both the old and the new test files pair) ?
You also need to add your new MIB files to this PR, that why tests cannot find the same results as on your dev machine.

**Comment by hrtrd** [comment]
yes, that's right, I completely forgot...

File: includes/definitions/discovery/bdcom.yaml (hunk 1 above)
Comment (PipoCanaja) [reply includes/definitions/discovery/bdcom.yaml:29]: Seems better now with mibs added. But still wondering if the bdpmCPUTotal5min should be kept aside of nmscardCPUUtilization or replaced. 
@hrtrd Are you sure nmscardCPUUtilization works on any device/version ? wouldn't it make sense to keep both in the yaml file?

**Review by murrant — APPROVED** [review]
LGTM, nice work.

**Pull request merged**